### ISSUE TITLE
Standardization of certifier report?

### ISSUE THREAD

**Issue by avneeshsingh** [w3c/publ-a11y#161]
This issue was raised by @folio @johnfoliot
John highlighted that EPUB accessibility metadata can point to certifier report, but there is no guidance for writing the report, and we should consider standardizing the report.

The other participants of the task force highlighted that it they kept away from this because the report can be used for various use cases, some may be using this report for validation purpose, some may be placing results of accessibility checking in this report and some may have some other use case for it.
The format of report would also differ according to the local requirements. For example EU countries may ask for some specific things in the report, United States may have different expectations like VPAT, India may have different expectations etc.

Therefore, we should have a good discussion on this issue tracker for figuring out the path ahead.

**Comment by mattgarrish** [comment]
I expect the best we could do here is recommend key information that should be in the report, sort of like what WCAG has for making claims: https://www.w3.org/TR/WCAG21/#conformance-claims

Given that we're probably something like 90% WCAG success criteria reporting and 10% EPUB, this may be the wrong venue for trying to find a universal reporting format.

The other part of the issue is where the reports should be hosted. We've tried the embedding approach with metadata records and if that's any precedent then expecting reading systems/vendors to present linked internal content may be a dead end. That leaves linking out to records. I've heard hesitation about sending users off vendor sites to external links, however, especially when those links are author provided (i.e., could present security issues). Some vendors probably will allow this, perhaps only for trusted publishers, but it seems strongly related to the work that's already going on for presenting the accessibility metadata. We should be able to find the answers there as part of how to present these links.

**Comment by RachelComerford** [comment]
While a conformance report will mean something to those working in the accessibility field, I'm worried that this would move us even further from supplying meaningful, easy to consume information for the average user. Already the metadata that we provide in many cases raises questions rather than supplying answers about the accessibility of the product, leading us to write longer and more detailed accessibility summaries so that we can introduce plain language explanations. 

In addition, this will likely become a deterrent for some publishers to update their backlist - developing a conformance report for hundreds or thousands of titles is financially not a viable ask and could quickly become an excuse to not improve any elements of ebooks in the backlist. 

Is the problem that this is trying to solve that there are unverified claims of accessible titles? If not, can I get a better idea of the why for this report?
